# Literature List

## MHC Binding Prediction


### IEDB posts:
* **2020:** [HLA allele frequencies and reference sets with maximal population coverage](https://help.iedb.org/hc/en-us/articles/114094151851-HLA-allele-frequencies-and-reference-sets-with-maximal-population-coverage)
* **2022**: [What allele types can I select for my MHC Class I and Class II predictions?](https://help.iedb.org/hc/en-us/articles/4416712893083-What-allele-types-can-I-select-for-my-MHC-Class-I-and-Class-II-predictions)
* **2025**: [Selecting thresholds (cut-offs) for MHC class I and II binding predictions](https://discuss.iedb.org/t/selecting-thresholds-cut-offs-for-mhc-class-i-and-ii-binding-predictions/182)

### NetMHCpan
* **Most recent publication:** Reynisson B., Alvarez B., Paul S., Peters B., Nielsen M. 2020. NetMHCpan-4.1 and NetMHCIIpan-4.0: improved predictions of MHC antigen presentation by concurrent motif deconvolution and integration of MS MHC eluted ligand data. Nucleic Acids Res. 48(W1):W449-W454.
PMID: 32406916

### NetMHCIIpan
* **Most recent publication:** Nilsson JB, Kaabinejadian S, Yari H, Kester MGD, van Balen P, Hildebrand WH, Nielsen M. 2023. Accurate prediction of HLA class II antigen presentation across all loci using tailored data acquisition and refined machine learning. Sci Adv.. 9(47):eadj6367. doi: 10.1126/sciadv.adj6367.
PMID: 38000035

*Older publications:*

### NetMHCpan
* Jurtz V, Paul S, Andreatta M, Marcatili P, Peters B, Nielsen M. 2017. NetMHCpan-4.0: Improved Peptide–MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data. The Journal of Immunology, 199(9):3360-3368.
PMID: 28978689

* Nielsen, M. and Andreatta, M., 2016. NetMHCpan-3.0; improved prediction of binding to MHC class I molecules integrating information from multiple receptor and peptide length datasets. Genome medicine, 8(1), p.33.
PMID: 27029192

* Hoof I, Peters B, Sidney J, Pedersen LE, Sette A, Lund O, Buus S, Nielsen M. 2009. NetMHCpan, a method for MHC class I binding prediction beyond humans. Immunogenetics 61:1-13.
PMID: 19002680

* Nielsen M, Lundegaard C, Blicher T, Lamberth K, Harndahl M, Justesen S, Roder G, Peters B, Sette A, Lund O, Buus S. 2007. NetMHCpan, a method for quantitative predictions of peptide binding to any HLA-A and -B locus protein of known sequence. PLoS ONE 2:e796.
PMID: 17726526 

### NetMHCIIpan
* Nilsson JB, Kaabinejadian S, Yari H, Peters B, Barra C, Gragert L, Hildebrand W, Nielsen M. 2023. Machine learning reveals limited contribution of trans-only encoded variants to the HLA-DQ immunopeptidome. Commun Biol. 6(1):442. doi: 10.1038/s42003-023-04749-7.
PMID: 37085710

* Kaabinejadian S, Barra C, Alvarez B, Yari H, Hildebrand WH, Nielsen M. 2022. Accurate MHC Motif Deconvolution of Immunopeptidomics Data Reveals a Significant Contribution of DRB3, 4 and 5 to the Total DR Immunopeptidome. Front Immunol. 13:835454.
PMID: 35154160

* Reynisson B., Alvarez B., Paul S., Peters B., Nielsen M. 2020. NetMHCpan-4.1 and NetMHCIIpan-4.0: improved predictions of MHC antigen presentation by concurrent motif deconvolution and integration of MS MHC eluted ligand data. Nucleic Acids Res. 48(W1):W449-W454.
PMID: 32406916

* Jensen KK, Andreatta M, Marcatili P, Buus S, Greenbaum JA, Yan Z, Sette A, Peters B, Nielsen M. 2018. Improved methods for predicting peptide binding affinity to MHC class II molecules. Immunology 154(3):394-406.
PMID: 29315598

* Andreatta M, Karosiene E, Rasmussen M, Stryhn A, Buus S, and Nielsen M. 2015. Accurate pan-specific prediction of peptide-MHC class II binding affinity with improved binding core identification. Immunogenetics.67(11-12):641-50.
PMID: 26416257


## HLA-I and HLA-II supertyping
* Wang, M., & Claesson, M. H. (2014). Classification of human leukocyte antigen (HLA) supertypes. Methods in molecular biology (Clifton, N.J.), 1184, 309–317. https://doi.org/10.1007/978-1-4939-1115-8_17
* Saha, I., Mazzocco, G., & Plewczynski, D. (2013). Consensus classification of human leukocyte antigen class II proteins. Immunogenetics, 65(2), 97–105. https://doi.org/10.1007/s00251-012-0665-6
* Shen, W.-J., Zhang, X., Zhang, S., Liu, C., & Cui, W. (2018). The Utility of Supertype Clustering in Prediction for Class II MHC-Peptide Binding. Molecules, 23(11), 3034. https://doi.org/10.3390/molecules23113034
* Greenbaum, J., Sidney, J., Chung, J., Brander, C., Peters, B., & Sette, A. (2011). Functional classification of class II human leukocyte antigen (HLA) molecules reveals seven different supertypes and a surprising degree of repertoire sharing across supertypes. Immunogenetics, 63(6), 325–335. https://doi.org/10.1007/s00251-011-0513-0


## Unipept
Link to web page: [https://unipept.ugent.be/publications](url)

Relevant publications (reverse-chronological):

* Tim Van Den Bossche, Pieter Verschaffelt, Tibo Vande Moortele, Peter Dawyndt, Lennart Martens, and Bart Mesuere
Biodiversity Analysis of Metaproteomics Samples with Unipept: A Comprehensive Tutorial
Protein Bioinformatics, 2024
doi.org/10.1007/978-1-0716-4007-4_11

* Tibo Vande Moortele, Bram Devlaminck, Simon Van de Vyver, Tim Van Den Bossche, Lennart Martens, Peter Dawyndt, Bart Mesuere, and Pieter Verschaffelt
Unipept in 2024: Expanding Metaproteomics Analysis with Support for Missed Cleavages, Semi-Tryptic and Non-Tryptic Peptides
Journal of Proteome Research, 2025
doi.org/10.1021/acs.jproteome.4c00848

* Pieter Verschaffelt, Alessandro Tanca, Marcello Abbondio, Tim Van Den Bossche, Tibo Vande Moortele, Peter Dawyndt, Lennart Martens, and Bart Mesuere
Unipept Desktop 2.0: Construction of Targeted Reference Protein Databases for Metaproteogenomics Analyses
Journal of Proteome Research, 2023
doi.org/10.1021/acs.jproteome.3c00091

* Pieter Verschaffelt, Tim Van Den Bossche, Lennart Martens, Peter Dawyndt, and Bart Mesuere
Unipept Desktop: a faster, more powerful metaproteomics results analysis tool
Journal of Proteome Research, 2021
doi.org/10.1021/acs.jproteome.0c00855

* Pieter Verschaffelt, Philippe Van Thienen, Tim Van Den Bossche, Felix Van der Jeugt, Caroline De Tender, Lennart Martens, Peter Dawyndt, and Bart Mesuere
Unipept CLI 2.0: adding support for visualizations and functional annotations
Bioinformatics, 2020
doi.org/10.1093/bioinformatics/btaa553

* Robbert Gurdeep Singh, Alessandro Tanca, Antonio Palomba, Felix Van der Jeugt, Pieter Verschaffelt, Sergio Uzzau, Lennart Martens, Peter Dawyndt, and Bart Mesuere
Unipept 4.0: functional analysis of metaproteome data
Journal of Proteome Research, 2019
doi.org/10.1021/acs.jproteome.8b00716

* Bart Mesuere, Felix Van der Jeugt, Toon Willems, Tom Naessens, Bart Devreese, Lennart Martens, and Peter Dawyndt
High-throughput metaproteomics data analysis with Unipept: a tutorial
Journal of Proteomics, 2018, 171, pp 11-22
doi.org/10.1016/j.jprot.2017.05.022

* Bart Mesuere, Felix Van der Jeugt, Bart Devreese, Peter Vandamme, and Peter Dawyndt
The unique peptidome: Taxon-specific tryptic peptides as biomarkers for targeted metaproteomics
Proteomics, 2016, 16 (17), pp 2313–2318
doi.org/10.1002/pmic.201600023

* Bart Mesuere, Toon Willems, Felix Van der Jeugt, Bart Devreese, Peter Vandamme, and Peter Dawyndt
Unipept Web Services for Metaproteomics Analysis
Bioinformatics, 2016, 32 (11), pp 1746-1748
doi.org/10.1093/bioinformatics/btw039

* Bart Mesuere, Griet Debyser, Maarten Aerts, Bart Devreese, Peter Vandamme, and Peter Dawyndt
The Unipept Metaproteomics Analysis Pipeline
Proteomics, 2015, 15 (8), pp 1437–1442
doi.org/10.1002/pmic.201400361

* Bart Mesuere, Bart Devreese, Griet Debyser, Maarten Aerts, Peter Vandamme, and Peter Dawyndt
Unipept: Tryptic Peptide-Based Biodiversity Analysis of Metaproteome Samples
J. Proteome Res., 2012, 11 (12), pp 5773–5780
doi.org/10.1021/pr300576s


## Mass spec-based immunopeptidomics

* **Kalaora, S. et al. Identification of bacteria-derived HLA-bound peptides in melanoma. Nature 592, 138-143
(2021).**

DOWNLOADED - MSBooster paper: https://www.nature.com/articles/s41467-023-40129-9

DOWNLOADED - 4. Declercq, A. et al. MS(2)Rescore: Data-Driven Rescoring Dramatically Boosts Immunopeptide Identification Rates. Mol Cell Proteomics 21, 100266 (2022).

DOWNLOADED  - 5. Wen, B., Li, K., Zhang, Y. & Zhang, B. Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis. Nat Commun 11, 1759 (2020).

DOWNLOADED - 6. Demichev, V., Messner, C.B., Vernardis, S.I., Lilley, K.S. & Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. Nat Methods 17, 41-44 (2020).

DOWNLOADED - 7. Frohlich, K. et al. Data-Independent Acquisition: A Milestone and Prospect in Clinical Mass Spectrometry-Based Proteomics. Mol Cell Proteomics 23, 100800 (2024).

DOWNLOADED - 8. Demichev, V. et al. dia-PASEF data analysis using FragPipe and DIA-NN for deep proteomics of low sample amounts. Nat Commun 13, 3944 (2022).

