# Literature list

## Topics
* Mass spectrometry (MS)-based immunopeptidomics
* Peptide identification
    - FragPipe
    - MSFragger
    - Percolator
    - Philosopher: ProteinProphet
    - MSBooster
* Peptide verification
    - PepQuery2
* Visualize proteomics data with PDViewer
* Taxonomy and Functions
    - Unipept
    - BLASTP
    - DIAMOND
    - MEGAN
* MHC binding prediction
    - IEDB
    - HLA class I and class II supertyping
    - NetMHCpan and NetMHCIIpan

### Status
* No checkbox - Keep for reference
‚ùì Might read.
‚ùó Plan to read.
üìñ Currently reading.
‚úÖ Have read. MM/DD/YY date finished.

**Datasets:**

‚úÖ 12/19/24 - Kalaora S, Nagler A, Nejman D, Alon M, Barbolin C, Barnea E, Ketelaars SLC, Cheng K, Vervier K, Shental N, Bussi Y, Rotkopf R, Levy R, Benedek G, Trabish S, Dadosh T, Levin-Zaidman S, Geller LT, Wang K, Greenberg P, Yagel G, Peri A, Fuks G, Bhardwaj N, Reuben A, Hermida L, Johnson SB, Galloway-Pe√±a JR, Shropshire WC, Bernatchez C, Haymaker C, Arora R, Roitman L, Eilam R, Weinberger A, Lotan-Pompan M, Lotem M, Admon A, Levin Y, Lawley TD, Adams DJ, Levesque MP, Besser MJ, Schachter J, Golani O, Segal E, Geva-Zatorsky N, Ruppin E, Kvistborg P, Peterson SN, Wargo JA, Straussman R, Samuels Y. Identification of bacteria-derived HLA-bound peptides in melanoma. Nature. 2021 Apr;592(7852):138-143. doi: 10.1038/s41586-021-03368-8. Epub 2021 Mar 17. PMID: 33731925; PMCID: PMC9717498.

‚úÖ 4/8/25 - Naghavian R, Faigle W, Oldrati P, Wang J, Toussaint NC, Qiu Y, Medici G, Wacker M, Freudenmann LK, Bont√© PE, Weller M, Regli L, Amigorena S, Rammensee HG, Walz JS, Brugger SD, Mohme M, Zhao Y, Sospedra M, Neidert MC, Martin R. Microbial peptides activate tumour-infiltrating lymphocytes in glioblastoma. Nature. 2023 May;617(7962):807-817. doi: 10.1038/s41586-023-06081-w. Epub 2023 May 17. PMID: 37198490; PMCID: PMC10208956.

## MS-based immunopeptidomics

‚úÖ 4/4/25 - ElAbd H, Franke A. Mass Spectrometry-Based Immunopeptidomics of Peptides Presented on Human Leukocyte Antigen Proteins. Methods Mol Biol. 2024;2758:425-443. doi: 10.1007/978-1-0716-3646-6_23. PMID: 38549028.

‚úÖ 2/14/25 - [Immunopeptidomics Overview](https://www.thermofisher.com/us/en/home/industrial/mass-spectrometry/mass-spectrometry-learning-center/mass-spectrometry-applications-area/immunopeptidomics.html)

* Zhang B, Bassani-Sternberg M. Current perspectives on mass spectrometry-based immunopeptidomics: the computational angle to tumor antigen discovery. J Immunother Cancer. 2023 Oct;11(10):e007073. doi: 10.1136/jitc-2023-007073. PMID: 37899131; PMCID: PMC10619091.

* Ahn R, Cui Y, White FM. Antigen discovery for the development of cancer immunotherapy. Semin Immunol. 2023 Mar;66:101733. doi: 10.1016/j.smim.2023.101733. Epub 2023 Feb 23. PMID: 36841147.

* Declercq, A. et al. MS(2)Rescore: Data-Driven Rescoring Dramatically Boosts Immunopeptide Identification Rates. Mol Cell Proteomics 21, 100266 (2022).

‚ùó Wen, B., Li, K., Zhang, Y. & Zhang, B. Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis. Nat Commun 11, 1759 (2020).

* Demichev, V., Messner, C.B., Vernardis, S.I., Lilley, K.S. & Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. Nat Methods 17, 41-44 (2020).

* Frohlich, K. et al. Data-Independent Acquisition: A Milestone and Prospect in Clinical Mass Spectrometry-Based Proteomics. Mol Cell Proteomics 23, 100800 (2024).

* Demichev, V. et al. dia-PASEF data analysis using FragPipe and DIA-NN for deep proteomics of low sample amounts. Nat Commun 13, 3944 (2022).

* Ritz D, Kinzi J, Neri D, Fugmann T. Data-Independent Acquisition of HLA Class I Peptidomes on the Q Exactive Mass Spectrometer Platform. Proteomics. 2017 Oct;17(19):10.1002/pmic.201700177. doi: 10.1002/pmic.201700177. PMID: 28834231; PMCID: PMC5846733.

* Pak H, Michaux J, Huber F, Chong C, Stevenson BJ, M√ºller M, Coukos G, Bassani-Sternberg M. Sensitive Immunopeptidomics by Leveraging Available Large-Scale Multi-HLA Spectral Libraries, Data-Independent Acquisition, and MS/MS Prediction. Mol Cell Proteomics. 2021;20:100080. doi: 10.1016/j.mcpro.2021.100080. Epub 2021 Apr 9. PMID: 33845167; PMCID: PMC8724634.

* Ferreira HJ, Stevenson BJ, Pak H, Yu F, Almeida Oliveira J, Huber F, Taillandier-Coindard M, Michaux J, Ricart-Altimiras E, Kraemer AI, Kandalaft LE, Speiser DE, Nesvizhskii AI, M√ºller M, Bassani-Sternberg M. Immunopeptidomics-based identification of naturally presented non-canonical circRNA-derived peptides. Nat Commun. 2024 Mar 15;15(1):2357. doi: 10.1038/s41467-024-46408-3. PMID: 38490980; PMCID: PMC10943130.

* Murphy JP, Konda P, Kowalewski DJ, Schuster H, Clements D, Kim Y, Cohen AM, Sharif T, Nielsen M, Stevanovic S, Lee PW, Gujar S. MHC-I Ligand Discovery Using Targeted Database Searches of Mass Spectrometry Data: Implications for T-Cell Immunotherapies. J Proteome Res. 2017 Apr 7;16(4):1806-1816. doi: 10.1021/acs.jproteome.6b00971. Epub 2017 Mar 21. PMID: 28244318.

* Nielsen M, Ternette N, Barra C. The interdependence of machine learning and LC-MS approaches for an unbiased understanding of the cellular immunopeptidome. Expert Rev Proteomics. 2022 Feb;19(2):77-88. doi: 10.1080/14789450.2022.2064278. Epub 2022 Apr 21. PMID: 35390265.

### Peptide identification using FragPipe

[https://fragpipe.nesvilab.org/](url)
[https://github.com/Nesvilab/FragPipe](url)

**Peptide-spectrum matching (PSMs) with MSFragger**

‚ùó Kong AT, Leprevost FV, Avtonomov DM, Mellacheruvu D, Nesvizhskii AI. MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics. Nat Methods. 2017 May;14(5):513-520. doi: 10.1038/nmeth.4256. Epub 2017 Apr 10. PMID: 28394336; PMCID: PMC5409104.

‚ùó Yu F, Teo GC, Kong AT, Haynes SE, Avtonomov DM, Geiszler DJ, Nesvizhskii AI. Identification of modified peptides using localization-aware open search. Nat Commun. 2020 Aug 13;11(1):4065. doi: 10.1038/s41467-020-17921-y. PMID: 32792501; PMCID: PMC7426425.

‚ùó Teo GC, Polasky DA, Yu F, Nesvizhskii AI. Fast Deisotoping Algorithm and Its Implementation in the MSFragger Search Engine. J Proteome Res. 2021 Jan 1;20(1):498-505. doi: 10.1021/acs.jproteome.0c00544. Epub 2020 Dec 17. PMID: 33332123; PMCID: PMC8864561.

**DDA quantification**

* Yu F, Haynes SE, Teo GC, Avtonomov DM, Polasky DA, Nesvizhskii AI. Fast Quantitative Analysis of timsTOF PASEF Data with MSFragger and IonQuant. Mol Cell Proteomics. 2020 Sep;19(9):1575-1585. doi: 10.1074/mcp.TIR120.002048. Epub 2020 Jul 2. PMID: 32616513; PMCID: PMC7996969.

* Yu, F., Haynes, S. E., & Nesvizhskii, A. I. (2021). IonQuant enables accurate and sensitive label-free quantification with FDR-controlled match-between-runs. Molecular & Cellular Proteomics, 20, 100077.

**PSM validation** 

Percolator:

‚ùó K√§ll L, Canterbury JD, Weston J, Noble WS, MacCoss MJ. Semi-supervised learning for peptide identification from shotgun proteomics datasets. Nat Methods. 2007 Nov;4(11):923-5. doi: 10.1038/nmeth1113. Epub 2007 Oct 21. PMID: 17952086.

PeptideProphet: *Note: not using for meta-iPep*

Keller A, Nesvizhskii AI, Kolker E, Aebersold R. Empirical statistical model to estimate the accuracy of peptide identifications made by MS/MS and database search. Anal Chem. 2002 Oct 15;74(20):5383-92. doi: 10.1021/ac025747h. PMID: 12403597.


**FDR filtering**

Philosopher:

‚ùó da Veiga Leprevost F, Haynes SE, Avtonomov DM, Chang HY, Shanmugam AK, Mellacheruvu D, Kong AT, Nesvizhskii AI. Philosopher: a versatile toolkit for shotgun proteomics data analysis. Nat Methods. 2020 Sep;17(9):869-870. doi: 10.1038/s41592-020-0912-y. PMID: 32669682; PMCID: PMC7509848.

**Protein inference**

ProteinProphet:

‚ùó Nesvizhskii AI, Keller A, Kolker E, Aebersold R. A statistical model for identifying proteins by tandem mass spectrometry. Anal Chem. 2003 Sep 1;75(17):4646-58. doi: 10.1021/ac0341261. PMID: 14632076.

**MSBooster**

‚ùó Yang KL, Yu F, Teo GC, Li K, Demichev V, Ralser M, Nesvizhskii AI. MSBooster: improving peptide identification rates using deep learning-based features. Nat Commun. 2023 Jul 27;14(1):4539. doi: 10.1038/s41467-023-40129-9. PMID: 37500632; PMCID: PMC10374903.

## NOTE: Kalaora et al. and Naghavian et al. use DDA data.

**Extract quantification from DIA data** *Note: not using for meta-iPep*

Demichev V, Messner CB, Vernardis SI, Lilley KS, Ralser M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. Nat Methods. 2020 Jan;17(1):41-44. doi: 10.1038/s41592-019-0638-x. Epub 2019 Nov 25. PMID: 31768060; PMCID: PMC6949130.

Demichev V, Szyrwiel L, Yu F, Teo GC, Rosenberger G, Niewienda A, Ludwig D, Decker J, Kaspar-Schoenefeld S, Lilley KS, M√ºlleder M, Nesvizhskii AI, Ralser M. dia-PASEF data analysis using FragPipe and DIA-NN for deep proteomics of low sample amounts. Nat Commun. 2022 Jul 8;13(1):3944. doi: 10.1038/s41467-022-31492-0. PMID: 35803928; PMCID: PMC9270362.

**Generate pseudo-MS/MS spectra from the DIA data** *Note: not using for meta-iPep*

Tsou CC, Avtonomov D, Larsen B, Tucholska M, Choi H, Gingras AC, Nesvizhskii AI. DIA-Umpire: comprehensive computational framework for data-independent acquisition proteomics. Nat Methods. 2015 Mar;12(3):258-64, 7 p following 264. doi: 10.1038/nmeth.3255. Epub 2015 Jan 19. PMID: 25599550; PMCID: PMC4399776.

**Identify peptides directly from DIA data** *Note: not using for meta-iPep*

Yu F, Teo GC, Kong AT, Fr√∂hlich K, Li GX, Demichev V, Nesvizhskii AI. Analysis of DIA proteomics data using MSFragger-DIA and FragPipe computational platform. Nat Commun. 2023 Jul 12;14(1):4154. doi: 10.1038/s41467-023-39869-5. PMID: 37438352; PMCID: PMC10338508.

## Peptide verification

**PepQuery2**

‚úÖ 4/14/25 - Wen B, Zhang B. PepQuery2 democratizes public MS proteomics data for rapid peptide searching. Nat Commun. 2023 Apr 18;14(1):2213. doi: 10.1038/s41467-023-37462-4. PMID: 37072382; PMCID: PMC10113256.

**PepQuery**

‚úÖ 4/14/25 - Wen B, Wang X, Zhang B. PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations. Genome Res. 2019 Mar;29(3):485-493. doi: 10.1101/gr.235028.118. Epub 2019 Jan 4. PMID: 30610011; PMCID: PMC6396417.


‚ùó **Also in MS-based immunopep section** Wen B, Li K, Zhang Y, Zhang B. Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis. Nat Commun. 2020 Apr 9;11(1):1759. doi: 10.1038/s41467-020-15456-w. PMID: 32273506; PMCID: PMC7145864.

## Visualize proteomics data using PDViewer

‚ùó Li K, Vaudel M, Zhang B, Ren Y, Wen B. PDV: an integrative proteomics data viewer. Bioinformatics. 2019 Apr 1;35(7):1249-1251. doi: 10.1093/bioinformatics/bty770. PMID: 30169737; PMCID: PMC6821182.

**Criteria for spectral quality**

üìñ start 4/14/25 - Rajczewski AT, Mehta S, Nguyen DDA, Gr√ºning B, Johnson JE, McGowan T, Griffin TJ, Jagtap PD. A rigorous evaluation of optimal peptide targets for MS-based clinical diagnostics of Coronavirus Disease 2019 (COVID-19). Clin Proteomics. 2021 May 10;18(1):15. doi: 10.1186/s12014-021-09321-1. PMID: 33971807; PMCID: PMC8107781.

‚ùó Jagtap PD, Johnson JE, Onsongo G, Sadler FW, Murray K, Wang Y, Shenykman GM, Bandhakavi S, Smith LM, Griffin TJ. Flexible and accessible workflows for improved proteogenomic analysis using the Galaxy framework. J Proteome Res. 2014 Dec 5;13(12):5898-908. doi: 10.1021/pr500812t. Epub 2014 Oct 23. PMID: 25301683; PMCID: PMC4261978.

## MHC Binding Prediction

### IEDB posts:

‚úÖ 2/27/25 - **2020:** [HLA allele frequencies and reference sets with maximal population coverage](https://help.iedb.org/hc/en-us/articles/114094151851-HLA-allele-frequencies-and-reference-sets-with-maximal-population-coverage)

‚úÖ 2/27/25 - **2022**: [What allele types can I select for my MHC Class I and Class II predictions?](https://help.iedb.org/hc/en-us/articles/4416712893083-What-allele-types-can-I-select-for-my-MHC-Class-I-and-Class-II-predictions)

‚úÖ 4/2/25 - **2025**: [Selecting thresholds (cut-offs) for MHC class I and II binding predictions](https://discuss.iedb.org/t/selecting-thresholds-cut-offs-for-mhc-class-i-and-ii-binding-predictions/182)

### HLA-I and HLA-II molecules

* Wang, M., & Claesson, M. H. (2014). Classification of human leukocyte antigen (HLA) supertypes. Methods in molecular biology (Clifton, N.J.), 1184, 309‚Äì317. https://doi.org/10.1007/978-1-4939-1115-8_17

**HLA class I supertypes**

‚ùìWeiskopf D, Angelo MA, de Azeredo EL, Sidney J, Greenbaum JA, Fernando AN, Broadwater A, Kolla RV, De Silva AD, de Silva AM, Mattia KA, Doranz BJ, Grey HM, Shresta S, Peters B, Sette A. Comprehensive analysis of dengue virus-specific responses supports an HLA-linked protective role for CD8+ T cells. Proc Natl Acad Sci U S A. 2013 May 28;110(22):E2046-53. doi: 10.1073/pnas.1305227110. Epub 2013 Apr 11. PMID: 23580623; PMCID: PMC3670335.


**HLA class II supertypes**

‚ùì Greenbaum, J., Sidney, J., Chung, J., Brander, C., Peters, B., & Sette, A. (2011). Functional classification of class II human leukocyte antigen (HLA) molecules reveals seven different supertypes and a surprising degree of repertoire sharing across supertypes. Immunogenetics, 63(6), 325‚Äì335. https://doi.org/10.1007/s00251-011-0513-0

‚ùì Shen, W.-J., Zhang, X., Zhang, S., Liu, C., & Cui, W. (2018). The Utility of Supertype Clustering in Prediction for Class II MHC-Peptide Binding. Molecules, 23(11), 3034. https://doi.org/10.3390/molecules23113034

‚ùì Saha, I., Mazzocco, G., & Plewczynski, D. (2013). Consensus classification of human leukocyte antigen class II proteins. Immunogenetics, 65(2), 97‚Äì105. https://doi.org/10.1007/s00251-012-0665-6



### NetMHCpan for HLA class I binding prediction

**These four (4) articles are to be cited during publication:**

‚úÖ 3/30/25 - NetMHCpan-4.1 and NetMHCIIpan-4.0: Improved predictions of MHC antigen presentation by concurrent motif deconvolution and integration of MS MHC eluted ligand data
Birkir Reynisson, Bruno Alvarez, Sinu Paul, Bjoern Peters and Morten Nielsen
Nucleic Acids Research, Volume 48, Issue W1, 02 July 2020, Pages W449‚ÄìW454, https://doi.org/10.1093/nar/gkaa379

‚ùó NetMHCpan-4.0: Improved Peptide MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data
Vanessa Jurtz, Sinu Paul, Massimo Andreatta, Paolo Marcatili, Bjoern Peters and Morten Nielsen
The Journal of Immunology (2017) ji1700893; DOI: 10.4049/jimmunol.1700893

‚úÖ 4/8/25 - NetMHCpan-3.0: improved prediction of binding to MHC class I molecules integrating information from multiple receptor and peptide length data sets
Morten Nielsen and Massimo Andreatta
Genome Medicine (2016): 8:33

‚úÖ 4/5/25 - NetMHCpan, a method for MHC class I binding prediction beyond humans
Ilka Hoof, Bjoern Peters, John Sidney, Lasse Eggers Pedersen, Ole Lund, Soren Buus, and Morten Nielsen
Immunogenetics 61.1 (2009): 1-13


### NetMHCIIpan for HLA class II binding prediction

**This one (1) article is to be cited during publication:** *Inverted peptide binders is new for version 4.3*

‚úÖ 3/30/25 - Accurate prediction of HLA class II antigen presentation across all loci using tailored data acquisition and refined machine learning
Jonas B. Nilsson, Saghar Kaabinejadian, Hooman Yari, Michel G. D. Kester, Peter van Balen, William H. Hildebrand and Morten Nielsen
Science Advances, 24 Nov 2023. https://www.science.org/doi/10.1126/sciadv.adj6367

**Inverted peptide articles:**

‚ùó S. Klobuch, J. J. Lim, P. van Balen, M. G. D. Kester, W. de Klerk, A. H. de Ru, C. R. Pothast, I. Jedema, J. W. Drijfhout, J. Rossjohn, H. H. Reid, P. A. van Veelen, J. H. F. Falkenburg, M. H. M. Heemskerk, Human T cells recognize HLA-DP‚Äìbound peptides in two orientations. Proc. Natl. Acad. Sci. U.S.A. 119, e2214331119 (2022).

‚úÖ 4/9/25 - J. Racle, P. Guillaume, J. Schmidt, J. Michaux, A. Larabi, K. Lau, M. A. S. Perez, G. Croce, R. Genolet, G. Coukos, V. Zoete, F. Pojer, M. Bassani-Sternberg, A. Harari, D. Gfeller, Machine learning predictions of MHC-II specificities reveal alternative binding mode of class II epitopes. Immunity 56, 1359‚Äì1375 (2023).

‚ùìRacle J, Michaux J, Rockinger GA, Arnaud M, Bobisse S, Chong C, Guillaume P, Coukos G, Harari A, Jandus C, Bassani-Sternberg M, Gfeller D. Robust prediction of HLA class II epitopes by deep motif deconvolution of immunopeptidomes. Nat Biotechnol. 2019 Nov;37(11):1283-1286. doi: 10.1038/s41587-019-0289-6. Epub 2019 Oct 14. PMID: 31611696.

**Context encoding articles:**

‚úÖ 3/30/25 - Barra C, Alvarez B, Paul S, Sette A, Peters B, Andreatta M, Buus S, Nielsen M. Footprints of antigen processing boost MHC class II natural ligand predictions. Genome Med. 2018 Nov 16;10(1):84. doi: 10.1186/s13073-018-0594-6. PMID: 30446001; PMCID: PMC6240193.

‚ùó Improved prediction of MHC II antigen presentation through integration and motif deconvolution of mass spectrometry MHC eluted ligand data.
Reynisson B, Barra C, Kaabinejadian S, Hildebrand WH, Peters B, Nielsen M
J Proteome Res 2020 Apr 30. doi: 10.1021/acs.jproteome.9b00874.
PubMed: 32308001


***Older publications:***

**NetMHCpan**

‚úÖ 3/5/25 - NetMHCpan, a Method for Quantitative Predictions of Peptide Binding to Any HLA-A and -B Locus Protein of Known Sequence.
Nielsen M, et al. (2007) PLoS ONE 2(8): e796. doi:10.1371/journal.pone.0000796

**NetMHC-4.0**

‚ùìAndreatta M, Nielsen M. Gapped sequence alignment using artificial neural networks: application to the MHC class I system. Bioinformatics. 2016 Feb 15;32(4):511-7. doi: 10.1093/bioinformatics/btv639. Epub 2015 Oct 29. PMID: 26515819; PMCID: PMC6402319.

**Non-canonical biners protruding at termini**

‚ùìCollins EJ, Garboczi DN, Wiley DC. Three-dimensional structure of a peptide extending from one end of a class I MHC binding site. Nature. 1994 Oct 13;371(6498):626-9. doi: 10.1038/371626a0. PMID: 7935798.

**Long peptides bulging out from MHC groove with canonical C and N**

‚ùìRist MJ, Theodossis A, Croft NP, Neller MA, Welland A, Chen Z, Sullivan LC, Burrows JM, Miles JJ, Brennan RM, Gras S, Khanna R, Brooks AG, McCluskey J, Purcell AW, Rossjohn J, Burrows SR. HLA peptide length preferences control CD8+ T cell responses. J Immunol. 2013 Jul 15;191(2):561-71. doi: 10.4049/jimmunol.1300292. Epub 2013 Jun 7. PMID: 23749632.

**NetMHCIIpan**

Machine learning reveals limited contribution of trans-only encoded variants to the HLA-DQ immunopeptidome
Jonas Birkelund Nilsson, Saghar Kaabinejadian, Hooman Yari, Bjoern Peters, Carolina Barra, Loren Gragert, William Hildebrand and Morten Nielsen
Communications Biology, 21 April 2023. https://doi.org/10.1038/s42003-023-04749-7

Accurate MHC Motif Deconvolution of immunopeptidomics data reveals high relevant contribution of DRB3, 4 and 5 to the total DR Immunopeptidome
Saghar Kaabinejadian, Carolina Barra, Bruno Alvarez, Hooman Yari, William Hildebrand, Morten Nielsen
Frontiers in Immunology 26 January 2022. Sec. Antigen Presenting Cell Biology, DOI: 10.3389/fimmu.2022.835454

‚ùóImproved methods for predicting peptide binding affinity to MHC class II molecules.
Jensen KK, Andreatta M, Marcatili P, Buus S, Greenbaum JA, Yan Z, Sette A, Peters B, Nielsen M.
Immunology. 2018 Jan 6. doi: 10.1111/imm.12889.
PubMed: 29315598

Accurate pan-specific prediction of peptide-MHC class II binding affinity with improved binding core identification
Andreatta M, Karosiene E, Rasmussen M, Stryhn A, Buus S, and Nielsen M
Immunogenetics (2015)
PubMed: 26416257

‚úÖ 3/6/25 - Quantitative predictions of peptide binding to any HLA-DR molecule of known sequence: NetMHCIIpan.
Nielsen M, et al. (2008) PLoS Comput Biol. Jul 4;4(7):e1000107.


**NetMHCII article:**

‚úÖ 3/5/25 - Prediction of MHC class II binding affinity using SMM-align, a novel stabilization matrix alignment method.
Nielsen, M, Lundegaard, C & Lund, O.
BMC Bioinformatics 8, 238 (2007). https://doi.org/10.1186/1471-2105-8-238


### ICERFIRE

‚ùìWan YR, Ko≈üaloƒülu-Yal√ßƒ±n Z, Peters B, Nielsen M. A large-scale study of peptide features defining immunogenicity of cancer neo-epitopes. NAR Cancer. 2024 Jan 29;6(1):zcae002. doi: 10.1093/narcan/zcae002. PMID: 38288446; PMCID: PMC10823584.

## Taxonomy and Functions

### Unipept
Link to web page: [https://unipept.ugent.be/publications](url)

Relevant publications (reverse-chronological):

‚ùó Tim Van Den Bossche, Pieter Verschaffelt, Tibo Vande Moortele, Peter Dawyndt, Lennart Martens, and Bart Mesuere
Biodiversity Analysis of Metaproteomics Samples with Unipept: A Comprehensive Tutorial
Protein Bioinformatics, 2024
doi.org/10.1007/978-1-0716-4007-4_11

‚ùó Tibo Vande Moortele, Bram Devlaminck, Simon Van de Vyver, Tim Van Den Bossche, Lennart Martens, Peter Dawyndt, Bart Mesuere, and Pieter Verschaffelt
Unipept in 2024: Expanding Metaproteomics Analysis with Support for Missed Cleavages, Semi-Tryptic and Non-Tryptic Peptides
Journal of Proteome Research, 2025
doi.org/10.1021/acs.jproteome.4c00848

* Pieter Verschaffelt, Alessandro Tanca, Marcello Abbondio, Tim Van Den Bossche, Tibo Vande Moortele, Peter Dawyndt, Lennart Martens, and Bart Mesuere
Unipept Desktop 2.0: Construction of Targeted Reference Protein Databases for Metaproteogenomics Analyses
Journal of Proteome Research, 2023
doi.org/10.1021/acs.jproteome.3c00091

* Pieter Verschaffelt, Tim Van Den Bossche, Lennart Martens, Peter Dawyndt, and Bart Mesuere
Unipept Desktop: a faster, more powerful metaproteomics results analysis tool
Journal of Proteome Research, 2021
doi.org/10.1021/acs.jproteome.0c00855

* Pieter Verschaffelt, Philippe Van Thienen, Tim Van Den Bossche, Felix Van der Jeugt, Caroline De Tender, Lennart Martens, Peter Dawyndt, and Bart Mesuere
Unipept CLI 2.0: adding support for visualizations and functional annotations
Bioinformatics, 2020
doi.org/10.1093/bioinformatics/btaa553

* Robbert Gurdeep Singh, Alessandro Tanca, Antonio Palomba, Felix Van der Jeugt, Pieter Verschaffelt, Sergio Uzzau, Lennart Martens, Peter Dawyndt, and Bart Mesuere
Unipept 4.0: functional analysis of metaproteome data
Journal of Proteome Research, 2019
doi.org/10.1021/acs.jproteome.8b00716

* Bart Mesuere, Felix Van der Jeugt, Toon Willems, Tom Naessens, Bart Devreese, Lennart Martens, and Peter Dawyndt
High-throughput metaproteomics data analysis with Unipept: a tutorial
Journal of Proteomics, 2018, 171, pp 11-22
doi.org/10.1016/j.jprot.2017.05.022

* Bart Mesuere, Felix Van der Jeugt, Bart Devreese, Peter Vandamme, and Peter Dawyndt
The unique peptidome: Taxon-specific tryptic peptides as biomarkers for targeted metaproteomics
Proteomics, 2016, 16 (17), pp 2313‚Äì2318
doi.org/10.1002/pmic.201600023

* Bart Mesuere, Toon Willems, Felix Van der Jeugt, Bart Devreese, Peter Vandamme, and Peter Dawyndt
Unipept Web Services for Metaproteomics Analysis
Bioinformatics, 2016, 32 (11), pp 1746-1748
doi.org/10.1093/bioinformatics/btw039

* Bart Mesuere, Griet Debyser, Maarten Aerts, Bart Devreese, Peter Vandamme, and Peter Dawyndt
The Unipept Metaproteomics Analysis Pipeline
Proteomics, 2015, 15 (8), pp 1437‚Äì1442
doi.org/10.1002/pmic.201400361

* Bart Mesuere, Bart Devreese, Griet Debyser, Maarten Aerts, Peter Vandamme, and Peter Dawyndt
Unipept: Tryptic Peptide-Based Biodiversity Analysis of Metaproteome Samples
J. Proteome Res., 2012, 11 (12), pp 5773‚Äì5780
doi.org/10.1021/pr300576s


### BLASTP

‚ùóAltschul, S.F., Gish, W., Miller, W., Myers, E.W., Lipman, D.J. (1990) ‚ÄúBasic local alignment search tool.‚Äù J. Mol. Biol. 215:403-410. PubMed

‚ùóAltschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller, W., Lipman, D.J. (1997) ‚ÄúGapped BLAST and PSI-BLAST: a new generation of protein database search programs.‚Äù Nucleic Acids Res. 25:3389-3402. PubMed


### DIAMOND
[https://github.com/bbuchfink/diamond](url)

When using the tool in published research, please cite: Buchfink B, Reuter K, Drost HG, "Sensitive protein alignments at tree-of-life scale using DIAMOND", Nature Methods 18, 366‚Äì368 (2021). doi:10.1038/s41592-021-01101-x

For sequence clustering: Buchfink B, Ashkenazy H, Reuter K, Kennedy JA, Drost HG, "Sensitive clustering of protein sequences at tree-of-life scale using DIAMOND DeepClust", bioRxiv 2023.01.24.525373; doi: https://doi.org/10.1101/2023.01.24.525373

Original publication to cite DIAMOND until v0.9.25: Buchfink B, Xie C, Huson DH, "Fast and sensitive protein alignment using DIAMOND", Nature Methods 12, 59-60 (2015). doi:10.1038/nmeth.3176

### MEGAN
[https://github.com/husonlab/megan-ce](url)

MEGAN Community Edition - Interactive exploration and analysis of large-scale microbiome sequencing data. Daniel H. Huson, Sina Beier, Isabell Flade, Anna Gorska, Mohamed El-Hadidi, Suparna Mitra, Hans-Joachim Ruscheweyh and Rewati Tappu. PLoS Computational Biology, 2016

Huson, D. H., Mitra, S., Ruscheweyh, H.-J., Weber, N., & Schuster, S. C. (2011). Integrative analysis of environmental sequences using MEGAN4. Genome Research, 21(9), 1552‚Äì1560. https://doi.org/10.1101/gr.120618.111
